cmd/main.go

```diff
@@ -60,7 +60,8 @@ func init() {
 func main() {
 	var (
 		// Environment variables
-		linodeToken string = os.Getenv("LINODE_TOKEN")
+		linodeToken    string = os.Getenv("LINODE_TOKEN")
+		linodeDNSToken string = os.Getenv("LINODE_DNS_TOKEN")
```

Comment:
nit: The type annotations here can be elided since `os.Getenv` returns a string.

```suggestion
		linodeToken    = os.Getenv("LINODE_TOKEN")
		linodeDNSToken = os.Getenv("LINODE_DNS_TOKEN")
```

Reply:
updated